heareval/predictions/task_predictions.py

```diff
@@ -782,14 +783,15 @@ def task_predictions_train(
             checkpoint_callback.best_model_score.detach().cpu().item(),
             json.dumps(best_postprocessing),
             "epoch %d" % epoch,
-            "time_in_min %.2f" % ((end - start) / 60),
+            "time_in_min %.2f" % time_in_min,
             json.dumps(hparams_to_json(predictor.hparams)),
             "\n\n\n",
         )
         sys.stdout.flush()
         return (
             checkpoint_callback.best_model_path,
             epoch,
+            time_in_min,
```

Comment:
This return is starting to get a bit gnarly. Does it have to be a list? getting to the point where returning a custom object or a dictionary would be nice

Reply:
@jorshi check latest push